File: fitrbackend/src/main/java/com/example/fitrbackend/dto/LocationResponse.java
package com.example.fitrbackend.dto;

import lombok.Getter;

public class LocationResponse {

    @Getter
    private final long id;
    @Getter
    private final long user_id;
    @Getter
    private final String name;
    @Getter
    private final String address;

    public LocationResponse(long id, long user_id, String name, String address) {
        this.id = id;
        this.user_id = user_id;
        this.name = name;
        this.address = address;
    }
}

File: fitrfrontend/fitrfrontend/API/LocationResponse.swift
//
//  LocationResponse.swift
//  fitrfrontend
//
//  Created by Ambrose Blay on 2/17/26.
//

import Foundation

struct LocationResponse: Codable, Identifiable {
    let id: Int64
    let userId: Int64
    let name: String
    let address: String
    
    enum CodingKeys: String, CodingKey {
        case id
        case userId = "user_id"
        case name
        case address
    }
}
